modules/model/StableDiffusionModel.py

```diff
@@ -179,36 +190,36 @@ def create_pipeline(self) -> DiffusionPipeline:
                 feature_extractor=None,
                 requires_safety_checker=False,
             )
-        else:
-            return StableDiffusionPipeline(
-                vae=self.vae,
-                text_encoder=self.text_encoder,
-                tokenizer=self.tokenizer,
-                unet=self.unet,
-                scheduler=self.noise_scheduler,
-                safety_checker=None,
-                feature_extractor=None,
-                requires_safety_checker=False,
-            )
+
```

Comment:
I'm not too sure about this one. It's true that the else branch isn't needed here, but it still makes the code clearer. With the old version, it was obvious that there are 3 branches that all depend on some conditions. In the formatted version, the last branch looks different from the other ones.

Reply:
These cases should then be written as a `match` rather than an `if`-`elif`-`else` cascade.